src/__tests__/demo.js

```diff
@@ -114,11 +114,11 @@ const fct2 = function(event, oldState, newState) {
   console.log('fct2');
   console.log('this', this);
   console.log('event', event);
-  console.log('oldState',oldState);
+  console.log('oldState', oldState);
   console.log('newState', newState);
 };
 
-window.adobeDataLayer.addEventListener('adobeDataLayer:change', fct2, {scope: 'future'});
+window.adobeDataLayer.addEventListener('adobeDataLayer:change', fct2, { scope: 'future' });
```

Comment:
Out of curiosity: it seems you are following spacing convention #1 below. Why this one and not others?
1. `{ scope: 'future' }`
2. `{ scope : 'future' }`
3. `{scope: 'future'}`

Reply:
This is the most common JS convention, I also use 1 and see that being used in most places.

Reply:
Interesting. In eslint the default is 3)
https://eslint.org/docs/rules/object-curly-spacing

There is probably an Adobe specific style guide that we could follow.